enatega-multivendor-web/src/components/Footer/newFooter/Links/SocialLinks.jsx

```
@@ -60,7 +61,15 @@ const SocialLinks = () => {
         }}
       >
         {socialLinks.map((item) => {
-          return <item.icon style={{ color: "#5AC12F" }} />;
+          return <Box key={item.name} onClick={
+            ()=>
+            {
+              navigate(item.url)
+            }
+          } >
+           <item.icon  style={{ color: "#5AC12F" , cursor:"pointer" }}  />;
+          </Box>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)